docs/polkadot-kusama-bridge-overview.md

```diff
@@ -0,0 +1,32 @@
+# Polkadot <> Kusama Bridge Overview
```

Comment:
To all reviewers: please review this document carefully - I'm describing my own impression of the bridge, which may be different from our actual plans.